code/modules/jobs/job_types/quartermaster.dm

```diff
@@ -36,6 +36,12 @@
 	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_BOLD_SELECT_TEXT | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
 	ignore_human_authority = TRUE
 
+	voice_of_god_power = 1.4 //Command staff has authority
```

Comment:
We shouldn't have to define this on every single head of staff, can you have voice of god check for this directly instead?

Reply:
this is turning into a refactor taking off all voice of god checks from jobs (command, mime, chaplain, curator) so gimmie a bit while i learn to do all this